src/async_std/mod.rs

```diff
@@ -1,8 +1,12 @@
 mod boxstream;
 mod handshake;
+mod tokio_compat;
```

Comment:
I'm wondering if this line also needs `#[cfg(feature = "tokio_compat")]` or not.